privacyidea/api/user.py

```diff
@@ -138,6 +144,7 @@ def delete_user(resolvername=None, username=None):
 @user_blueprint.route('/', methods=['POST'])
 @admin_required
 @prepolicy(check_base_action, request, ACTION.ADDUSER)
+@event("user_create", request, g)
```

Comment:
Should we call this ``user_add`` instead of ``user_create``, to have naming consistent with the ``adduser`` policy?

Reply:
you're right. Will do.